test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/TransactionTests.cs

```diff
@@ -27,11 +30,27 @@ public TransactionTests(HttpIntegrationTestFixture fixture)
 
         [Fact]
         [Trait(Traits.Priority, Priority.One)]
-        public async Task WhenSubmittingATransaction_GivenAProperBundle_ThenNotSupportedIsReturned()
+        public async Task GivenABundleWithVersionedReference_WhenSubmittingATransaction_ThenResolvedReferenceIsVersionSpecific()
         {
-            FhirException ex = await Assert.ThrowsAsync<FhirException>(() => Client.PostBundleAsync(Samples.GetDefaultTransaction().ToPoco<Bundle>()));
+            var requestBundle1 = Samples.GetJsonSample("Bundle-TransactionWithVersionSpecificReference");
+            var requestBundle = requestBundle1.ToPoco<Bundle>();
```

Comment:
nit: this line can be merged with the previous line
 var requestBundle = Samples.GetJsonSample("Bundle-TransactionWithVersionSpecificReference").ToPoco<Bundle>();